packages/web-components/src/checkbox/checkbox.styles.ts

```diff
@@ -74,51 +79,68 @@ export const checkboxStyles: (context: ElementDefinitionContext, definition: Che
       line-height: ${typeRampBaseLineHeight};
     }
 
-    .checked-indicator {
+    slot[name='checked-indicator'],
+    slot[name='indeterminate-indicator'] {
```

Comment:
I think it's beneficial to be specific, though it seems unlikely, it's possible other slots could be added some day. I think this would provide more explanation for anyone needing to override the style on this component.

I'm looking into future cleanup of this sort of issue though, and I think there's another way to resolve it I'll be working on proving out soon.